src/Orchard.Web/Modules/Orchard.MediaProcessing/Services/ImageProfileManager.cs

```diff
@@ -64,6 +64,18 @@ public class ImageProfileManager : IImageProfileManager {
             var filePath = _fileNameProvider.GetFileName(profileName, System.Web.HttpUtility.UrlDecode(path));
             bool process = false;
 
+            //after reboot the app cache is empty so we reload the image in the cache if it exists in the _Profiles folder
+            if (string.IsNullOrEmpty(filePath))
+            {
+                var filterContext = new FilterContext { FilePath = _storageProvider.Combine("_Profiles", FormatProfilePath(profileName, System.Web.HttpUtility.UrlDecode(path))) };

```

Comment:
You are right. It was because I was scary of breaking something of a code I didn't know so I copied code from other part without understanding it very well. (Bad decission).